examples/connector/python/build.sh

```diff
@@ -6,13 +6,18 @@ source connector_run/bin/activate
 pip install -r requirements.txt
 
 # copying protos present in the root of directory to `protos` folder
-mkdir -p protos
-cp ../../../*.proto protos/
+mkdir -p proto
+for file in ../../../*v2.proto; do
+    file_name=$(basename "$file")
+    new_file="proto/${file_name%_v2.proto}.proto"
+    cp "$file" "$new_file"
+done
+
 # Generates the required gRPC Python files using protos into `sdk_pb2` folder
 mkdir -p sdk_pb2
 python -m grpc_tools.protoc \
-       --proto_path=./protos/ \
+       --proto_path=./proto/ \
```

Comment:
why renaming this folder?

Reply:
I have updated name to be consistent with other language examples. In java and golang we use proto.